src/pages/Landing/components/StudentTicket/History/History.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-nested-ternary */
+import React, { useState } from 'react'
+import attachment from '../../../../../assets/images/attachment.png'
+import styles from './History.module.scss'
+import head from './mockData'
+import deleteIcon from '../../../../../components/icons/icons-svg/delete.svg'
+
+const History = () => {
+  return (
+    <div className={styles.History}>
+      <h4 className={styles.HistoryText}>Report History</h4>
+
+      <div className={styles.HistoryTable}>
+        <div className={styles.Header}>
+          <h2 className={styles.Instructor}>Instructor</h2>
+          <h2 className={styles.IssueTitle}>Issue Title</h2>
+          <h2 className={styles.Priority}>Priority</h2>
+          <h2 className={styles.Description}>Description</h2>
+          <h2 className={styles.Status}>Status</h2>
+        </div>
+
+        {head.map(item => {
+          return (
+            <div className={styles.Detail}>
+              <h2 className={styles.Instructor}>{item.Instructor}</h2>
+              <h2 className={styles.IssueTitle}>{item.IssueTitle}</h2>
+              {item.Priority === 'Low' ? (
```

Comment:
Create a enum for Priorities to prevent typo errors
export const Priorites= {
    Low: "Low",
    Medium: "Medium",
    High: "High",
}

Reply:
how i use enum ...i never use it before....i google about it but I could not understand how to implement .

Reply:
If you use `item.Priority === 'Low'`, if you have some typos like `low`  with `Low`. It will be wrong. Instead of using this `item.Priority === Priorites.Low`. It will better, we will not be scared about typo errors.

Reply:
i dont get your point....please exlpain the code ....what you want me to use ...if typos so how I can use it

Reply:
`Priorites.Low === 'Low'` Do you know about using object in javascript? just change `Low` to `Priorites.Low`. How about if you type `item.Priority === 'low'` instead of `item.Priority === 'Low'`. It will be wrong right. That is a typo?

Reply:
ok i change the code as you said item.Priority === 'low' to item.Priority === Priority.Low .........but its not working .......i don't think there will be any typo error come we used priority button low ,high and medium and and these event values will be fire always when we press button.so there is not input thing so how typo error came.